### DISCUSSION TITLE
Permission to see a form

### DISCUSSION THREAD

**Discussion by jaquelinedealmeida** [Budibase/budibase#13600] [Help] [answered]
Hi, everyone. I am new in the Budibase and building a extern form with Rest API. 
When I put the fields from queries to create a form, there isn't permission to insert the values on a form.

Is it because the form is from Rest API or it's default from budibase?

**Comment by Abd-Atif** [comment, marked as answer]
Hi @jaquelinedealmeida

some steps to troubleshoot:

Check API Permissions: Make sure the REST API you're using allows data to be written, not just read. You may need specific permissions to perform write operations.
Form Setup: Verify that your form is correctly configured to send data to the API. Ensure the form's action points to the correct API endpoint and uses the right method (like POST).
User Roles: Look at the permissions for the user role associated with the form. Ensure they have the rights to make changes via the API.
API Configuration: Double-check your API integration settings in Budibase, especially headers and authentication tokens, to ensure your requests are authorized.

**Reply by jaquelinedealmeida** [reply]
Thanks. I managed to solve it.
